libs9s/s9sglobal.h

```diff
@@ -37,6 +37,12 @@ namespace S9s
         PublicDnsName,
         PrivateDnsName,
     };
+
+    enum Redirect
```

Comment:
I propose to use `enum class` and use it like `Redirect::Allow` or `Redirect::Deny`.
No duplication will be required.

Reply:
Yeah, that is nice. But this is good enough too.